platform/fakes/fake_platform.go

```diff
@@ -197,8 +199,14 @@ func (p *FakePlatform) SetupSSH(publicKey, username string) error {
 	return p.SetupSSHErr
 }
 
-func (p *FakePlatform) SetUserPassword(user, encryptedPwd string) (err error) {
-	p.UserPasswords[user] = encryptedPwd
+func (p *FakePlatform) SetUserPassword(user string, encryptedPwd string) (err error) {
+	if user == boshsettings.VCAPUsername {
+		p.UserPassword = encryptedPwd
+	}
```

Comment:
I added a new line there, as discussed with @maximilien and @cfdreddbot , we need to escalate the issue about disabling root login to high level operation or management team.